lms/djangoapps/instructor/views/gradebook_api.py

```diff
@@ -143,8 +142,8 @@ def get_student_statistics(student_email, course_id):
     }
     response = send_edeos_api_request(**edeos_post_data)
     tokens = 'n/a'
-    try:
-        tokens = response[student_email][0]["tokens"]
-    except (KeyError, TypeError) as err:
-        log.exception("Edeos student statistics call failed. {}".format(err.message))
+    students = filter(lambda students: students.get(student_email), response)
+    
+    if students:
+        return students[0][student_email][0]['tokens'] if students[0][student_email][0] else tokens
```

Comment:
maybe it will be better:
`for student in response:
    if student_email in student.keys:
        tokens = student[student_email][0]['tokens']
        return tokens
return tokens`